src/@orb.yml

```diff
@@ -7,6 +7,6 @@ display:
   source_url: https://github.com/noosenergy/noos-circleci-orb
 
 orbs:
-  aws-cli: circleci/aws-cli@3.1.3
+  aws-cli: circleci/aws-cli@4.0.0
```

Comment:
⚠️ 

Maybe worth to check if Orbs API's haven't changed as a sanitary check?

Reply:
The API's did change but the changes don't impact our orb